examples/window.rs

```diff
@@ -983,6 +1063,14 @@ const KEY_BINDINGS: &[Binding<&'static str>] = &[
     // C.
     Binding::new("C", ModifiersState::CONTROL, Action::NextCursor),
     Binding::new("C", ModifiersState::ALT, Action::NextCustomCursor),
+    #[cfg(web_platform)]
+    Binding::new("C", ModifiersState::SHIFT, Action::UrlCustomCursor),
+    #[cfg(web_platform)]
+    Binding::new(
+        "C",
+        ModifiersState::CONTROL.union(ModifiersState::SHIFT),
```

Comment:
If you want to combine, use `|`, it's more clear.

Reply:
Unfortunately that's not possible in a `const` context.